examples/crewai/python/src/honcho_crewai/exceptions.py

```
@@ -0,0 +1,14 @@
+"""
+Exception classes for CrewAI integration.
+"""
+
+
+class HonchoDependencyError(ImportError):
+    """Raised when required CrewAI dependencies are not installed."""
+
+    def __init__(self, framework: str, install_command: str):
```

Ruff (0.14.5)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)